### PR TITLE
Tek830 a 41 implement improvements of layout and UI

### PR DESCRIPTION
**Pull request by erbergli**
Adds a button for activating HÄRMAPA, a list for displaying scheduled events and updates the overall look and layout of the GUI.
This also changes the code to use customTkinter, which is a more modern and better looking extension of regular tkinter. Both frameworks are mostly interchangable, meaning that it wasn't that hard to change between them, which is why it is included in this PR.
image

### CODE REVIEW

File: View/AptLayout.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ def __init__(self, parent):
         for room, coordinate in self.room_coordinations.items():
             self.darken_rooms(coordinate)
         self.display_layout(parent)
+
```
Comment (filipkauffeldt) [inline View/AptLayout.py:46]: Remove new line

File: View/activation_button.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    def __init__(self, parent):
+        super().__init__(master=parent)
+        
+        self._obeservers = []
+        
+        self._activated = False
+        
+        self._btn = ctk.CTkButton(self,
+                                  text="Activate HÄRMAPA", 
+                                  fg_color="#2C2C2C",
```
Comment (filipkauffeldt) [inline View/activation_button.py:17]: Do we want these to be constants, like some other colours we have?

File: View/clock_widget.py [hunk 3]
```diff
@@ -19,15 +19,15 @@ def __init__(self, parent, controller : TimeSlider):
...
-        self._time_label = tk.Label(self, text=f"{self._current_time}", font=("Helvetica", 18))
-        self._time_label.grid(row=0, column=0, padx=10, pady=20)  # Using grid to place the label
-
-        self.pack(pady=20)
-            
+        self._time_label = ctk.CTkLabel(self, text=f"{self._current_time}", font=("Helvetica", 30), text_color="white")
+        self._time_label.grid(row=0, column=0, padx=10)
+        self._time_label.configure(fg_color="#0057AD")
+        
+        self.configure(fg_color="#0057AD")
```
Comment (filipkauffeldt) [inline View/clock_widget.py]: Could this colour be a constant?

File: View/schedule_list.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        
+        # create variable text for frame title
+        self._title_lbl_text_var : tk.StringVar = tk.StringVar()
+        self._title_lbl_text_var.set(self._deactivated_str)
+        
+        #Init title label for scrollable frame
+        self._title_lbl_frame = ctk.CTkFrame(self, 
+                                             width=self._width, 
+                                             height=30,
+                                             fg_color = "#ebe8e8",
```
Comment (filipkauffeldt) [inline View/schedule_list.py]: Could the colour be a constant?

**Review by filipkauffeldt — APPROVED** [review]

File: View/schedule_list.py (hunk 4 above)
Comment (erbergli) [reply View/schedule_list.py]: Yes of course

File: View/clock_widget.py (hunk 3 above)
Comment (erbergli) [reply View/clock_widget.py]: Yes of course

File: View/activation_button.py (hunk 2 above)
Comment (erbergli) [reply View/activation_button.py:17]: Yes I thought about it as well, but did not implement it for some reason

**Pull request merged**